src/App.tsx

```diff
@@ -40,9 +41,26 @@ export default function App() : JSX.Element {
     Roboto_500Medium,
   });
 
-  if (!fontsLoaded) {
-    return <AppLoading />;
-  }
+  useEffect(() => {
+    async function prepare(): Promise<void> {
+      try {
+        // Keep the splash screen visible while we fetch resources
+        // Make any API calls you need to do here
+        // Artificially delay for two seconds to simulate a slow loading
+        // experience. Please remove this when you're ready to approve the PR! @AndreLaranjeira
+        await new Promise((resolve) => setTimeout(resolve, 2000));
```

Comment:
@AndreLaranjeira remember to remove the fake call, it's here just to help you to see the splash screen

Reply:
Actually, I think this fake call doesn't generate any delay. The setTimeout function tries to execute what you pass it and aborts after the timeout, but since the method passed is resolve, which is not asynchronous, I think this promise just resolves immediately.

Regardless, I was able to test it out. It works very well.